components/telegram_bot_api/telegram_bot_api.app.js

```diff
@@ -58,6 +74,161 @@ module.exports = {
         "Only emit events for the selected update types.",
       options: updateTypes,
     },
+    chatId: {
+      type: "string",
+      label: "Chat ID",
+      description: "Enter the unique identifier for the target chat or username of the target channel (in the format `@channelusername` or `@supergroupusername`).",
+    },
+    text: {
+      type: "string",
+      label: "Text",
+      description: "Enter or map the message text to send.",
+      optional: true,
+    },
+    parse_mode: {
+      type: "string",
+      label: "Parse Mode",
+      description: "Select [MarkdownV2-style](https://core.telegram.org/bots/api#markdownv2-style), [HTML-style](https://core.telegram.org/bots/api#html-style), or [Markdown-style](https://core.telegram.org/bots/api#markdown-style) of the text if you want Telegram apps to show bold, italic, fixed-width text or inline URLs in your bot's message.",
+      options: [
+        "MarkdownV2",
+        "HTML",
+        "Markdown",
+      ],
+      optional: true,
+    },
+    disable_notification: {
+      type: "boolean",
+      label: "Disable Notifications",
+      description: "Choose if to send the message silently. iOS users will not receive a notification, Android users will receive a notification with no sound.",
+      optional: true,
+    },
+    disable_web_page_preview: {
+      type: "boolean",
+      label: "Disable Link Previews",
+      description: "Choose if to disable link previews for links in this message.",
+      optional: true,
+    },
+    reply_to_message_id: {
+      type: "string",
+      label: "Original Message ID",
+      description: "Enter the ID of the original message.",
+      optional: true,
+    },
+    reply_markup: {
+      type: "string",
+      label: "Reply Markup",
+      description: "Enter additional interface options that are a JSON-serialized object including an inline keyboard, a custom reply keyboard, instructions to remove the reply keyboard or instructions to force a reply from the user, e.g. `{\"inline_keyboard\":[[{\"text\":\"Some button text 2\",\"url\":\"https://botpress.org\"}]]}` or `{\"keyboard\":[[\"Yes\",\"No\"],[\"Maybe\"]]}`. Note: keyboard cannot be used with channels.",
```

Comment:
Seems like there is a lot of options here, can you add like a link so the user can have more idea about these properties?

Reply:
Added a link to the docs in the `description` of `reply_markup`:

https://github.com/PipedreamHQ/pipedream/blob/c80b5afcf9851debaa7cab8efab46a3619aeaa1f/components/telegram_bot_api/telegram_bot_api.app.mjs#L80-L87